transformer_engine/jax/csrc/modules.cpp

```diff
@@ -90,12 +91,15 @@ pybind11::bytes PackCustomCallFusedAttnDescriptor(
 }
 
 bool IsFusedAttnKernelAvailable() {
-#if (CUDNN_VERSION >= 8901)
-    auto major = cudaDevicePropertiesManager::Instance().GetMajor();
-    // Fused attention requires at least Ampere
-    return major >= 8;
-#else
+#if (CUDNN_VERSION < 8901)
     return false;
+#else
+    const int sm_arch = cuda::sm_arch();
+#if (CUDNN_VERSION >= 8903)
```

Comment:
Can we keep this cuDNN version check in `nvte_get_fused_attn_backend` so TE-PyTorch could benefit from it as well?

Reply:
Sure, I updated it.